pdftotree/TreeExtract.py

```diff
@@ -402,42 +402,72 @@ def get_html_others(self, tag: str, box: List[float], page_num: int) -> Element:
                 word_element.appendChild(self.doc.createTextNode(text))
         return element
 
-    def get_html_table(self, table, page_num) -> Element:
-        table_str = [str(i) for i in table]
+    def get_html_table(self, table: List[float], page_num) -> Optional[Element]:
+        """Recognize a table using tabula and return a DOM element.
+
+        :param table: bbox for a table (top,left,bottom,right)
+        :param page_num: 1-based page number
+        :return: DOM element for a table
+        """
+        self.log.debug(f"Calling tabula at page: {page_num} and area: {table}.")
```

Comment:
Can we use global logging instead of logging in the class?

Reply:
Great idea! Resolved by 2b68c4b9d7aad0013e40d93e9180d2f7d00191ff.